README.md

```diff
@@ -24,3 +30,5 @@ Pull these files into the Maxim 1.2.0 SDK that supports the MAX32625.
     * 't' - triggered mode
 
 All modes emit full-precision version of the data displayed on the LCD.  Units aren't emitted, but are consistant with the LCD.  Labels aren't emitted but the CSV values are in the same order as is displayed on the LCD.  Units and labels are static.

```

Comment:
Correct spelling of "consistent".
Replace contraction aren't with long form "are not".